lib/stormpath-sdk/oauth/password_grant.rb

```diff
@@ -1,20 +1,23 @@
 module Stormpath
   module Oauth
     class PasswordGrant < Stormpath::Resource::Base
-      prop_accessor :grant_type, :username, :password
+      prop_accessor :grant_type, :username, :password, :organization_name_key
 
       def form_properties
-        {
+        hash = {
           grant_type: grant_type,
           username: username,
           password: password
         }
+        hash[:organizationNameKey] = organization_name_key unless organization_name_key.nil?
+        hash
       end
```

Comment:
@DamirSvrtan instructions added to readme and rewrote the form_properties method.